lyricsbot/domains/songlyrics/songlyrics.py

```diff
@@ -0,0 +1,51 @@
+"""
+Getting song lyrics via users' data.
+"""
+from bs4 import BeautifulSoup
+
+import requests
```

Comment:
Make properly python imports. Look at [this](https://github.com/reddit/reddit/wiki/PythonImportGuidelines).

Reply:
Done.